colibre-zooms/auto_plotter/gas_fraction_plots.yml

```diff
@@ -160,14 +161,15 @@ hi_frac_func_ssfr:
   metadata:
     title: sSFR-HI Fraction
     section: Cold Gas Data Comparison
-    caption: Galaxy HI mass over stellar mass as a function of specific star formation rate in 0.2 dex bins, measured in 100 kpc apertures
+    caption: Galaxy HI mass over stellar mass as a function of specific star formation rate in 0.2 dex bins, measured in 100 kpc apertures. XGASS (Catinella+18) galaxies are selected with to have a flat $M_\star$ distribution in the range $10^9 \; {\rm M_\odot} > M_\star > 10^{10} \; {\rm M_\odot}$, and a higher normalised uniform in the range $10^{10} \; {\rm M_\odot} > M_\star > 10^{11} \; {\rm M_\odot}$.
```

Comment:
"XGASS (Catinella+18) galaxies are selected with to have": I think the "with" has to go?